deploy/playbooks/roles/common/templates/prod.py.j2

```diff
@@ -1,6 +1,6 @@
 # {{ ansible_managed }}
 from prado.hooks import error
-from prod_api_creds import api_key, api_user
+from config.prod_api_creds import api_key, api_user
```

Comment:
I don't think you'd need ``config`` here, the relative import should work. It would be ``configs`` anyway though, right?